app/v1/models/models.py

```diff
@@ -0,0 +1,263 @@
+import datetime
+# =====================================DbUsers==============================================
+
+class DbUsers:
+    def __init__(self):
+        self.all_users = dict()
+        self.all_emails = list()
+        self.id = 1
+
+    def add_user(self, email, username, password, address):
+        if email not in self.all_emails:
+            try:
+                self.all_users[username]
+            except KeyError:
```

Comment:
I thought we talked about this. The except should be reporting errors. But I see you still have actual code implemented there.